ci-operator/step-registry/ipi/aws/pre/edge-zones/opt-in/ipi-aws-pre-edge-zones-opt-in-commands.sh

```diff
@@ -82,27 +82,14 @@ function select_option() {
             echo "Selecting random zone with type: ${zone_type}";
             select_and_opt_in_zone "${zone_type}";
             ;;
-        "") echo "Ignoring single zone EDGE_ZONE_TYPE=${zone_type}" ;;
```

Comment:
@mtulio  Please double check if this line can be removed. Per my understanding, except `local-zone` and `wavelength-zone`, all other values should be treated as invalid.

Reply:
@yunjiang29 the changes looks good to me. We can now a single flow/option.